assets/images/content-icons/developer-community-icon.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="48px" height="42px" viewBox="0 0 48 42" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 61.1 (101010) - https://sketch.com -->
+    <title>004F276C-10EC-4D48-966E-C7469D756DC7</title>
+    <desc>Created with sketchtool.</desc>
+    <g id="Pages" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Home-page-01" transform="translate(-916.000000, -717.000000)" fill="#D9725C" fill-rule="nonzero">
+            <g id="Group-7-Copy" transform="translate(886.000000, 684.000000)">
+                <g id="developer-community-icon-48x48" transform="translate(30.000000, 33.000000)">
+                    <path d="M47,0 L17,0 C16.4477153,0 16,0.44771525 16,1 L16,10 L1,10 C0.44771525,10 0,10.4477153 0,11 L0,33 C0,33.5522847 0.44771525,34 1,34 L4,34 L4,41 C3.99989823,41.4044291 4.24340926,41.7691037 4.617,41.924 C4.73833175,41.9745857 4.86854611,42 5,42 C5.26519481,42 5.51950727,41.8945494 5.707,41.707 L13.414,34 L31,34 C31.5522847,34 32,33.5522847 32,33 L32,24 L34.586,24 L42.293,31.707 C42.4804927,31.8945494 42.7348052,31.9999434 43,32 C43.1314539,32.0004246 43.2616682,31.9745857 43.383,31.924 C43.7565907,31.7691037 44.0001018,31.4044291 44,31 L44,24 L47,24 C47.5522847,24 48,23.5522847 48,23 L48,1 C48,0.44771525 47.5522847,0 47,0 Z M30,32 L13,32 C12.7348052,32.0000566 12.4804927,32.1054506 12.293,32.293 L6,38.586 L6,33 C6,32.4477153 5.55228475,32 5,32 L2,32 L2,12 L16,12 L16,23 C16,23.5522847 16.4477153,24 17,24 L30,24 L30,32 Z M46,22 L43,22 C42.4477153,22 42,22.4477153 42,23 L42,28.586 L35.707,22.293 C35.5195073,22.1054506 35.2651948,22.0000566 35,22 L18,22 L18,2 L46,2 L46,22 Z" id="Shape"></path>
+                    <path d="M40,14 C41.1045695,14 42,13.1045695 42,12 C42,10.8954305 41.1045695,10 40,10 C38.8954305,10 38,10.8954305 38,12 C38,13.1045695 38.8954305,14 40,14 Z" id="Path"></path>
+                    <path d="M32,14 C33.1045695,14 34,13.1045695 34,12 C34,10.8954305 33.1045695,10 32,10 C30.8954305,10 30,10.8954305 30,12 C30,13.1045695 30.8954305,14 32,14 Z" id="Path"></path>
+                    <path d="M24,14 C25.1045695,14 26,13.1045695 26,12 C26,10.8954305 25.1045695,10 24,10 C22.8954305,10 22,10.8954305 22,12 C22,13.1045695 22.8954305,14 24,14 Z" id="Path"></path>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
This is probably already on your radar, but just a reminder that depending on the merge order between this and Spencer's PR that changes the images folder structure, there might be some conflicts or weirdness here

Reply:
Thanks for that reminder still! Re-arranged the image location [here](https://github.com/Dwolla/dev-portal/pull/66/commits/c1e7b5d84d64b30443817ba6b7cd1efee3ca5847).